.eslintrc-react-test

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
Ah okay.  I didn't understand its purpose - I was reading it as "this is the the eslintrc file that I'm testing for React right now".  Question withdrawn!